class-es-wp-tax-query.php

```diff
@@ -75,8 +75,8 @@ public function get_dsl( $es_query ) {
 					break;
 
 				default :
-					$terms = array_map( 'absint', $query['terms'] );
-					$current_filter = $es_query::dsl_terms( $es_query->tax_map( $query['taxonomy'], 'term_id' ), $query['terms'], $filter_options );
+					$terms = array_map( 'absint', array_map( 'strtolower', array_values( $query['terms'] ) ) );
```

Comment:
I don't think we need `strtolower()` here because we are already calling `absint()` on the term value.